iOS Client/Store/MBPodcastsStore.swift

```diff
@@ -135,18 +135,16 @@ class MBPodcastsStore: PodcastsRepository {
             return []
         }
         
-        return visibilitySettings
-            .keys
-            .filter { visibilitySettings[$0] ?? false }
-            .compactMap {
-                if let stream = PodcastStream(rawValue: $0) {
-                    return PodcastFilterOption.stream(stream)
-                } else if $0 == PodcastFilterOption.downloaded.key {
-                    return PodcastFilterOption.downloaded
-                } else {
-                    return nil
-                }
-            }
+        var enabledStreams = self.streams
+            .filter { visibilitySettings[$0.rawValue] ?? true } // default to enabling the stream toggles
+            .map { PodcastFilterOption.stream($0) }
+        
+        // default to disabling the 'downloaded-only' toggle
+        if visibilitySettings[PodcastFilterOption.downloaded.key] ?? false {
+            enabledStreams += [PodcastFilterOption.downloaded]
+        }
+        
+        return enabledStreams
```

Comment:
and refined this logic to default stream toggles to be `on` (and default the downloaded-only toggle to false). This way, newly added podcasts will show up without needing to be explicitly toggled on by the user